### PR TITLE
Docs/prepare release v2.1.0

### PR DESCRIPTION
**Pull request by nutellinoit**

### CODE REVIEW

File: katalog/configs/bases/coredns/dashboards/coredns.json [hunk 1]
```diff
@@ -1919,4 +1919,4 @@
   "timezone": "browser",
   "title": "CoreDNS",
   "version": 0
-}
+}
```
Comment (ralgozino) [inline katalog/configs/bases/coredns/dashboards/coredns.json:1922]: nitpick: missing new line at the end of file

**Review by ralgozino — CHANGES_REQUESTED** [review]
minor tweaks.

this PR is also doing a merge of `main` into the thanos PR, just double-checking that this is intended.

File: docs/releases/v2.1.0.md [hunk 2]
```diff
@@ -28,4 +29,6 @@ TBD
 
 To upgrade this core module from `v2.0.1` to `v2.1.0`, execute the following:
 
-TBD
+```bash
+kustomize build <your-project-path> | kubectl apply -f -
+```
```
[the remark this answers is not in the document]
Comment (nutellinoit) [reply docs/releases/v2.1.0.md:34]: Yep, I think that for now we can also skip this guide and add it later

**Review by ralgozino — APPROVED** [review]
lgtm

**Comment by Deepzima** [comment]
LGTM

**Review by Deepzima — APPROVED** [review]

**Pull request merged**